lib/matplotlib/artist.py

```diff
@@ -1510,7 +1510,7 @@ def setp(obj, *args, **kwargs):
 
 
 def kwdoc(artist):
-    """
+    r"""
```

Comment:
My question is, why is there a `\s` in the first place? Doesn't seem like we need the backslash at all?

Reply:
Inline markup, like the cross-reference, cannot end in the middle of a word.